plugins/modules/cloud/misc/terraform.py

```diff
@@ -93,6 +93,24 @@
     type: list
     elements: path
     version_added: '0.2.0'
+  plan_id:
+    description:
+      - A plan_id linked to a temporary plan file created if no plan_path
+        is specified.
+    required: false
```

Comment:
You can leave `required: false` away (it's the default). But what you must do is add `type: xxx`, where `xxx` is replaced by the correct type.